[INFO ] Logging to: /rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/outdir_train_configuration/RandomForest_Iris_scenario/log-run2.txt
[INFO ] Version of SMAC is v2.10.03-master-778 (3ee628ef9bf2), running on OpenJDK 64-Bit Server VM (1.8.0_442) and Linux 4.18.0-553.36.1.el8_10.x86_64 (amd64) 
[INFO ] Call String: smac --scenario-file Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/RandomForest_Iris_scenario.txt --seed 2 
[INFO ] SMAC started at: Apr 14, 2025 8:16:31 AM. Minimizing mean quality.
[INFO ] First incumbent: config 1 (internal ID: 0x000B), with mean quality: -0.9666666666666667; estimate based on 1 runs.
[INFO ] Sample call for new incumbent config 1 (internal ID: 0x000B): 
cd /rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example; /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 8792767 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.0' -min_samples_leaf '1' -min_samples_split '2' -min_weight_fraction_leaf '0.0' -n_estimators '100' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:16:50_182494_4006115000.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:16:50_182494_4006115000.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:16:50_182494_4006115000.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "gini", "max_depth_type": "None", "max_features_float": "0.5877139399025175", "max_features_type": "float", "max_leaf_nodes": "545", "max_leaf_nodes_type": "int", "max_samples_type": "None", "min_impurity_decrease": "0.24923987790177465", "min_samples_leaf": "1", "min_samples_split": "52", "min_weight_fraction_leaf": "0.32709269497250804", "n_estimators": "82", "oob_score": "False", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "13794010", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 13794010 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_float '0.5877139399025175' -max_features_type 'float' -max_leaf_nodes '545' -max_leaf_nodes_type 'int' -max_samples_type 'None' -min_impurity_decrease '0.24923987790177465' -min_samples_leaf '1' -min_samples_split '52' -min_weight_fraction_leaf '0.32709269497250804' -n_estimators '82' -oob_score 'False'  
[WARN ] Detected CRASHED run. The Solution Quality of CRASHED runs will be transformed to the MAX(quality, 1.0E9), to disable this use --transform-crashed-quality false to change the value use --transform-crashed-quality-value
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:05_182710_2558767300.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:05_182710_2558767300.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:05_182710_2558767300.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "log_loss", "max_depth_type": "None", "max_features_float": "0.3225253570857677", "max_features_type": "float", "max_leaf_nodes": "826", "max_leaf_nodes_type": "int", "max_samples": "0.32877074929063793", "max_samples_type": "float", "min_impurity_decrease": "0.1597443821201594", "min_samples_leaf": "24", "min_samples_split": "24", "min_weight_fraction_leaf": "0.46224628613811847", "n_estimators": "260", "oob_score": "False", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "13794010", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 13794010 -bootstrap 'True' -criterion 'log_loss' -max_depth_type 'None' -max_features_float '0.3225253570857677' -max_features_type 'float' -max_leaf_nodes '826' -max_leaf_nodes_type 'int' -max_samples '0.32877074929063793' -max_samples_type 'float' -min_impurity_decrease '0.1597443821201594' -min_samples_leaf '24' -min_samples_split '24' -min_weight_fraction_leaf '0.46224628613811847' -n_estimators '260' -oob_score 'False'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:21_182784_1623583181.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:21_182784_1623583181.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:21_182784_1623583181.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "entropy", "max_depth_type": "None", "max_features_float": "0.1678789016747193", "max_features_type": "float", "max_leaf_nodes_type": "None", "max_samples_type": "None", "min_impurity_decrease": "0.4011549023532129", "min_samples_leaf": "14", "min_samples_split": "37", "min_weight_fraction_leaf": "0.1862917324133434", "n_estimators": "38", "oob_score": "True", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "12632380", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 12632380 -bootstrap 'True' -criterion 'entropy' -max_depth_type 'None' -max_features_float '0.1678789016747193' -max_features_type 'float' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.4011549023532129' -min_samples_leaf '14' -min_samples_split '37' -min_weight_fraction_leaf '0.1862917324133434' -n_estimators '38' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:35_182868_712136389.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:35_182868_712136389.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:35_182868_712136389.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_float": "0.6541265776675875", "max_features_type": "float", "max_leaf_nodes": "471", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.1636503437824895", "min_samples_leaf": "61", "min_samples_split": "22", "min_weight_fraction_leaf": "0.11876163559064201", "n_estimators": "288", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "11794476", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 11794476 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_float '0.6541265776675875' -max_features_type 'float' -max_leaf_nodes '471' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.1636503437824895' -min_samples_leaf '61' -min_samples_split '22' -min_weight_fraction_leaf '0.11876163559064201' -n_estimators '288'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:50_182945_2292437559.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:50_182945_2292437559.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:50_182945_2292437559.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_special": "sqrt", "max_features_type": "special", "max_leaf_nodes": "464", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.1440022449951575", "min_samples_leaf": "35", "min_samples_split": "26", "min_weight_fraction_leaf": "0.3523320198867802", "n_estimators": "151", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "12632380", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 12632380 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes '464' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.1440022449951575' -min_samples_leaf '35' -min_samples_split '26' -min_weight_fraction_leaf '0.3523320198867802' -n_estimators '151'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:05_183164_2537307060.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:05_183164_2537307060.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:05_183164_2537307060.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "entropy", "max_depth": "43", "max_depth_type": "int", "max_features_float": "0.005254712622516644", "max_features_type": "float", "max_leaf_nodes": "442", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.4198833043922767", "min_samples_leaf": "88", "min_samples_split": "17", "min_weight_fraction_leaf": "0.24969575537993233", "n_estimators": "458", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "12632380", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 12632380 -bootstrap 'False' -criterion 'entropy' -max_depth '43' -max_depth_type 'int' -max_features_float '0.005254712622516644' -max_features_type 'float' -max_leaf_nodes '442' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.4198833043922767' -min_samples_leaf '88' -min_samples_split '17' -min_weight_fraction_leaf '0.24969575537993233' -n_estimators '458'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:35_183324_872474499.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:35_183324_872474499.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:35_183324_872474499.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "log_loss", "max_depth_type": "None", "max_features_float": "0.5831169778580023", "max_features_type": "float", "max_leaf_nodes_type": "None", "max_samples_type": "None", "min_impurity_decrease": "0.08945432970147071", "min_samples_leaf": "62", "min_samples_split": "89", "min_weight_fraction_leaf": "0.0979268510031765", "n_estimators": "283", "oob_score": "False", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "8720656", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 8720656 -bootstrap 'True' -criterion 'log_loss' -max_depth_type 'None' -max_features_float '0.5831169778580023' -max_features_type 'float' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.08945432970147071' -min_samples_leaf '62' -min_samples_split '89' -min_weight_fraction_leaf '0.0979268510031765' -n_estimators '283' -oob_score 'False'  
[INFO ] Updated estimated mean quality of the same incumbent: -0.96; estimate now based on 10 runs.
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:50_183402_3364059210.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:50_183402_3364059210.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:50_183402_3364059210.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "gini", "max_depth_type": "None", "max_features_special": "None", "max_features_type": "special", "max_leaf_nodes_type": "None", "max_samples": "0.09234710661066714", "max_samples_type": "float", "min_impurity_decrease": "0.4140565974552512", "min_samples_leaf": "24", "min_samples_split": "1", "min_weight_fraction_leaf": "0.07001574019504092", "n_estimators": "293", "oob_score": "False", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "8720656", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 8720656 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_special 'None' -max_features_type 'special' -max_leaf_nodes_type 'None' -max_samples '0.09234710661066714' -max_samples_type 'float' -min_impurity_decrease '0.4140565974552512' -min_samples_leaf '24' -min_samples_split '1' -min_weight_fraction_leaf '0.07001574019504092' -n_estimators '293' -oob_score 'False'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:19:05_183626_2574429564.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:19:05_183626_2574429564.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:19:05_183626_2574429564.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "log_loss", "max_depth_type": "None", "max_features_float": "0.5050284976068342", "max_features_type": "float", "max_leaf_nodes_type": "None", "max_samples_type": "None", "min_impurity_decrease": "0.2935459793493308", "min_samples_leaf": "41", "min_samples_split": "25", "min_weight_fraction_leaf": "0.27127377487302673", "n_estimators": "479", "oob_score": "True", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "1865372", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 1865372 -bootstrap 'True' -criterion 'log_loss' -max_depth_type 'None' -max_features_float '0.5050284976068342' -max_features_type 'float' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.2935459793493308' -min_samples_leaf '41' -min_samples_split '25' -min_weight_fraction_leaf '0.27127377487302673' -n_estimators '479' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:19:35_183788_3310307127.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:19:35_183788_3310307127.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:19:35_183788_3310307127.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_float": "0.24150108395825476", "max_features_type": "float", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.10881396615101702", "min_samples_leaf": "13", "min_samples_split": "61", "min_weight_fraction_leaf": "0.058115930951153194", "n_estimators": "405", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "8792767", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 8792767 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_float '0.24150108395825476' -max_features_type 'float' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.10881396615101702' -min_samples_leaf '13' -min_samples_split '61' -min_weight_fraction_leaf '0.058115930951153194' -n_estimators '405'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:20:20_184231_2440346474.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:20:20_184231_2440346474.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:20:20_184231_2440346474.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "entropy", "max_depth_type": "None", "max_features_float": "0.707465765251343", "max_features_type": "float", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.18675397299534596", "min_samples_leaf": "69", "min_samples_split": "82", "min_weight_fraction_leaf": "0.4946331087566143", "n_estimators": "350", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "8720656", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 8720656 -bootstrap 'False' -criterion 'entropy' -max_depth_type 'None' -max_features_float '0.707465765251343' -max_features_type 'float' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.18675397299534596' -min_samples_leaf '69' -min_samples_split '82' -min_weight_fraction_leaf '0.4946331087566143' -n_estimators '350'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:21:35_184788_612536857.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:21:35_184788_612536857.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:21:35_184788_612536857.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth": "48", "max_depth_type": "int", "max_features_float": "0.7755070347090761", "max_features_type": "float", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.11306284092768354", "min_samples_leaf": "99", "min_samples_split": "4", "min_weight_fraction_leaf": "0.13063062991648944", "n_estimators": "39", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "8792767", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 8792767 -bootstrap 'False' -criterion 'gini' -max_depth '48' -max_depth_type 'int' -max_features_float '0.7755070347090761' -max_features_type 'float' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.11306284092768354' -min_samples_leaf '99' -min_samples_split '4' -min_weight_fraction_leaf '0.13063062991648944' -n_estimators '39'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:21:50_184868_3227194972.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:21:50_184868_3227194972.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:21:50_184868_3227194972.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_float": "0.10103841562575933", "max_features_type": "float", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.04387595020268026", "min_samples_leaf": "78", "min_samples_split": "25", "min_weight_fraction_leaf": "0.01442349237479329", "n_estimators": "218", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "2248053", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 2248053 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_float '0.10103841562575933' -max_features_type 'float' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.04387595020268026' -min_samples_leaf '78' -min_samples_split '25' -min_weight_fraction_leaf '0.01442349237479329' -n_estimators '218'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:05_185083_3537126501.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:05_185083_3537126501.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:05_185083_3537126501.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "log_loss", "max_depth": "75", "max_depth_type": "int", "max_features_float": "0.8423477643997689", "max_features_type": "float", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.07625217534807499", "min_samples_leaf": "31", "min_samples_split": "93", "min_weight_fraction_leaf": "0.16465090106088975", "n_estimators": "37", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "12632380", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 12632380 -bootstrap 'False' -criterion 'log_loss' -max_depth '75' -max_depth_type 'int' -max_features_float '0.8423477643997689' -max_features_type 'float' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.07625217534807499' -min_samples_leaf '31' -min_samples_split '93' -min_weight_fraction_leaf '0.16465090106088975' -n_estimators '37'  
[INFO ] Updated estimated mean quality of the same incumbent: -0.9616666666666667; estimate now based on 20 runs.
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:20_185157_417571140.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:20_185157_417571140.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:20_185157_417571140.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth": "22", "max_depth_type": "int", "max_features_special": "sqrt", "max_features_type": "special", "max_leaf_nodes": "918", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.038587400367021185", "min_samples_leaf": "11", "min_samples_split": "25", "min_weight_fraction_leaf": "0.06703066993615375", "n_estimators": "360", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "12892141", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 12892141 -bootstrap 'False' -criterion 'gini' -max_depth '22' -max_depth_type 'int' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes '918' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.038587400367021185' -min_samples_leaf '11' -min_samples_split '25' -min_weight_fraction_leaf '0.06703066993615375' -n_estimators '360'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:34_185244_1448416580.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:34_185244_1448416580.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:34_185244_1448416580.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth": "13", "max_depth_type": "int", "max_features_special": "sqrt", "max_features_type": "special", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.21843739134213064", "min_samples_leaf": "84", "min_samples_split": "66", "min_weight_fraction_leaf": "0.15416508152441794", "n_estimators": "2", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "12892141", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 12892141 -bootstrap 'False' -criterion 'gini' -max_depth '13' -max_depth_type 'int' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.21843739134213064' -min_samples_leaf '84' -min_samples_split '66' -min_weight_fraction_leaf '0.15416508152441794' -n_estimators '2'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:19_185627_1243468538.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:19_185627_1243468538.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:19_185627_1243468538.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "log_loss", "max_depth_type": "None", "max_features_float": "0.9339649154409295", "max_features_type": "float", "max_leaf_nodes": "860", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.03692799082387305", "min_samples_leaf": "91", "min_samples_split": "36", "min_weight_fraction_leaf": "0.49710261410501955", "n_estimators": "70", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "1865372", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 1865372 -bootstrap 'False' -criterion 'log_loss' -max_depth_type 'None' -max_features_float '0.9339649154409295' -max_features_type 'float' -max_leaf_nodes '860' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.03692799082387305' -min_samples_leaf '91' -min_samples_split '36' -min_weight_fraction_leaf '0.49710261410501955' -n_estimators '70'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:33_185711_1371768331.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:33_185711_1371768331.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:33_185711_1371768331.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "log_loss", "max_depth_type": "None", "max_features_float": "0.12475588599038778", "max_features_type": "float", "max_leaf_nodes_type": "None", "max_samples": "0.4133775197631875", "max_samples_type": "float", "min_impurity_decrease": "0.04609684317864354", "min_samples_leaf": "98", "min_samples_split": "88", "min_weight_fraction_leaf": "0.15893860555151879", "n_estimators": "327", "oob_score": "False", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "8720656", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 8720656 -bootstrap 'True' -criterion 'log_loss' -max_depth_type 'None' -max_features_float '0.12475588599038778' -max_features_type 'float' -max_leaf_nodes_type 'None' -max_samples '0.4133775197631875' -max_samples_type 'float' -min_impurity_decrease '0.04609684317864354' -min_samples_leaf '98' -min_samples_split '88' -min_weight_fraction_leaf '0.15893860555151879' -n_estimators '327' -oob_score 'False'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:48_185786_565859334.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:48_185786_565859334.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:48_185786_565859334.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "entropy", "max_depth_type": "None", "max_features_float": "0.3415899230922854", "max_features_type": "float", "max_leaf_nodes": "425", "max_leaf_nodes_type": "int", "max_samples_type": "None", "min_impurity_decrease": "0.0369195304158233", "min_samples_leaf": "89", "min_samples_split": "83", "min_weight_fraction_leaf": "0.2517155363947597", "n_estimators": "126", "oob_score": "True", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "2248053", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 2248053 -bootstrap 'True' -criterion 'entropy' -max_depth_type 'None' -max_features_float '0.3415899230922854' -max_features_type 'float' -max_leaf_nodes '425' -max_leaf_nodes_type 'int' -max_samples_type 'None' -min_impurity_decrease '0.0369195304158233' -min_samples_leaf '89' -min_samples_split '83' -min_weight_fraction_leaf '0.2517155363947597' -n_estimators '126' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:24:05_186014_2972687677.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:24:05_186014_2972687677.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:24:05_186014_2972687677.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "log_loss", "max_depth": "84", "max_depth_type": "int", "max_features_float": "0.4102699702206981", "max_features_type": "float", "max_leaf_nodes": "568", "max_leaf_nodes_type": "int", "max_samples": "0.22069643735779543", "max_samples_type": "float", "min_impurity_decrease": "0.2593189983920268", "min_samples_leaf": "53", "min_samples_split": "39", "min_weight_fraction_leaf": "0.03365915802119007", "n_estimators": "210", "oob_score": "False", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "3971554", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 3971554 -bootstrap 'True' -criterion 'log_loss' -max_depth '84' -max_depth_type 'int' -max_features_float '0.4102699702206981' -max_features_type 'float' -max_leaf_nodes '568' -max_leaf_nodes_type 'int' -max_samples '0.22069643735779543' -max_samples_type 'float' -min_impurity_decrease '0.2593189983920268' -min_samples_leaf '53' -min_samples_split '39' -min_weight_fraction_leaf '0.03365915802119007' -n_estimators '210' -oob_score 'False'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:24:35_186170_882251685.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:24:35_186170_882251685.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:24:35_186170_882251685.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_special": "sqrt", "max_features_type": "special", "max_leaf_nodes": "597", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.38302152420047414", "min_samples_leaf": "25", "min_samples_split": "99", "min_weight_fraction_leaf": "0.3501776943019631", "n_estimators": "220", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "12089153", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 12089153 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes '597' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.38302152420047414' -min_samples_leaf '25' -min_samples_split '99' -min_weight_fraction_leaf '0.3501776943019631' -n_estimators '220'  
[INFO ] Updated estimated mean quality of the same incumbent: -0.9533333333333334; estimate now based on 30 runs.
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:24:50_186244_3930831282.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:24:50_186244_3930831282.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:24:50_186244_3930831282.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "gini", "max_depth_type": "None", "max_features_float": "0.04981648291319518", "max_features_type": "float", "max_leaf_nodes_type": "None", "max_samples_type": "None", "min_impurity_decrease": "0.02430859076362496", "min_samples_leaf": "38", "min_samples_split": "48", "min_weight_fraction_leaf": "0.006539708209397033", "n_estimators": "228", "oob_score": "False", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "15313704", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 15313704 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_float '0.04981648291319518' -max_features_type 'float' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.02430859076362496' -min_samples_leaf '38' -min_samples_split '48' -min_weight_fraction_leaf '0.006539708209397033' -n_estimators '228' -oob_score 'False'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:04_186604_1055515988.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:04_186604_1055515988.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:04_186604_1055515988.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_special": "None", "max_features_type": "special", "max_leaf_nodes": "693", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.3427749301946459", "min_samples_leaf": "15", "min_samples_split": "9", "min_weight_fraction_leaf": "0.1812109269224736", "n_estimators": "311", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "13688243", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 13688243 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_special 'None' -max_features_type 'special' -max_leaf_nodes '693' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.3427749301946459' -min_samples_leaf '15' -min_samples_split '9' -min_weight_fraction_leaf '0.1812109269224736' -n_estimators '311'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:19_186683_3838736601.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:19_186683_3838736601.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:19_186683_3838736601.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "log_loss", "max_depth_type": "None", "max_features_float": "0.2099524358573237", "max_features_type": "float", "max_leaf_nodes": "601", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.1768077945137247", "min_samples_leaf": "26", "min_samples_split": "31", "min_weight_fraction_leaf": "0.0042541080783955465", "n_estimators": "115", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "4309701", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 4309701 -bootstrap 'False' -criterion 'log_loss' -max_depth_type 'None' -max_features_float '0.2099524358573237' -max_features_type 'float' -max_leaf_nodes '601' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.1768077945137247' -min_samples_leaf '26' -min_samples_split '31' -min_weight_fraction_leaf '0.0042541080783955465' -n_estimators '115'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:34_186769_3424097189.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:34_186769_3424097189.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:34_186769_3424097189.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "entropy", "max_depth": "58", "max_depth_type": "int", "max_features_special": "None", "max_features_type": "special", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.10639850821930935", "min_samples_leaf": "22", "min_samples_split": "76", "min_weight_fraction_leaf": "0.36635508914006915", "n_estimators": "339", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "3971554", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 3971554 -bootstrap 'False' -criterion 'entropy' -max_depth '58' -max_depth_type 'int' -max_features_special 'None' -max_features_type 'special' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.10639850821930935' -min_samples_leaf '22' -min_samples_split '76' -min_weight_fraction_leaf '0.36635508914006915' -n_estimators '339'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:49_186842_382704799.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:49_186842_382704799.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:49_186842_382704799.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "log_loss", "max_depth": "13", "max_depth_type": "int", "max_features_special": "sqrt", "max_features_type": "special", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.238120420554138", "min_samples_leaf": "39", "min_samples_split": "83", "min_weight_fraction_leaf": "0.001764993853692276", "n_estimators": "272", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "1865372", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 1865372 -bootstrap 'False' -criterion 'log_loss' -max_depth '13' -max_depth_type 'int' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.238120420554138' -min_samples_leaf '39' -min_samples_split '83' -min_weight_fraction_leaf '0.001764993853692276' -n_estimators '272'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:26:03_187062_480672658.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:26:03_187062_480672658.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:26:03_187062_480672658.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "log_loss", "max_depth": "99", "max_depth_type": "int", "max_features_float": "0.702482061525599", "max_features_type": "float", "max_leaf_nodes": "156", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.012176977996275873", "min_samples_leaf": "26", "min_samples_split": "4", "min_weight_fraction_leaf": "0.3229729718638581", "n_estimators": "336", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "933951", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 933951 -bootstrap 'False' -criterion 'log_loss' -max_depth '99' -max_depth_type 'int' -max_features_float '0.702482061525599' -max_features_type 'float' -max_leaf_nodes '156' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.012176977996275873' -min_samples_leaf '26' -min_samples_split '4' -min_weight_fraction_leaf '0.3229729718638581' -n_estimators '336'  
[INFO ] 
=======================================================================================
SMAC has finished. Reason: wall-clock time limit (601.61 s) has been reached.   
Total number of runs performed: 80, total configurations tried: 36.
Total CPU time used: 56 s, total wallclock time used: 602 s.
SMAC's final incumbent: config 1 (internal ID: 0x000B), with estimated mean quality: -0.9518518518518518, based on 36 run(s) on 1 training instance(s).
Sample call for this final incumbent:
cd /rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example; /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 6980494 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.0' -min_samples_leaf '1' -min_samples_split '2' -min_weight_fraction_leaf '0.0' -n_estimators '100' -oob_score 'True'
=======================================================================================
[INFO ] 
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Additional information about run 2 in:/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/outdir_train_configuration/RandomForest_Iris_scenario
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Organising done! See Output/Performance_Data/performance_data.csv for results.
